packages/core/src/forms/control-action/control-action.element.scss

```diff
@@ -35,14 +32,6 @@ slot {
   pointer-events: none;
 }
 
-::slotted(cds-icon[shape='exclamation-circle']),
-::slotted(cds-icon[shape='check-circle']),
-::slotted(cds-icon[shape='info-circle']) {
-  @include equilateral($cds-token-space-size-9);
-  margin: calc(-1 * #{$cds-token-space-size-3});
-  margin-bottom: 0;
-}
-
```

Comment:
With the whitespace offset removal in the cds-icon itself the styles become simpler as we dont have to compensate for the misalignment within the svg.